packages/orchestration/test/cosmos-interchain-service.test.ts

```diff
@@ -174,10 +176,7 @@ test.serial('makeAccount returns an IcaAccountKit', async t => {
   );
 
   const delegateResp = await E(account).executeEncodedTx([delegateMsg]);
-  t.deepEqual(
-    tryDecodeResponse(delegateResp, MsgDelegateResponse.fromProtoMsg),
-    {},
-  );
+  t.deepEqual(tryDecodeResponse(delegateResp, MsgDelegateResponse), {});
```

Comment:
I don't understand this change.

at a glance, it looks like a breaking API change

Reply:
ok, so it is some API evolution.

bonus points for a mention in **Documentation Considerations**

Reply:
Mentioned.  Bonus points collected!